src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+public class StringCalculator {
+    public static final int ZERO = 0;
+    public static final String DELIMITER = ",|:";
+
+    public static int splitAndSum(String text){
+        if(isBlank(text))
```

Comment:
```suggestion
        if(isBlank(text)) {
            ...
        }
```
한줄이라도 블록으로 감싸주는 것이 좋습니다.
나중에 기능이 추가된다면 다시 블록을 추가해야하는 것도 일이고 
놓치게 된다면 원치않는 동작도 발생할 수 있습니다